packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/common_ui_components/app_error_model_widget.dart

```diff
@@ -0,0 +1,48 @@
+{{> licence.dart }}
+
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:rx_bloc/rx_bloc.dart';
+
+import '../../lib_router/blocs/router_bloc.dart';
+import '../extensions/error_model_translations.dart';
+import '../models/errors/error_model.dart';
+
+typedef ErrorStateCallback<BlocType extends RxBlocTypeBase> = Stream<ErrorModel>
+    Function(BlocType bloc);
+
+class AppErrorModelWidget<BlocType extends RxBlocTypeBase>
```

Comment:
@PVGeorgievPrime, I guess here you mean mod**a**l, not mod**e**l

Reply:
Renamed